src/Searchbar.vue

```diff
@@ -15,38 +15,95 @@ export default {
   },
   computed: {
     primitiveData() {
+      function allMatch(info, regexes) {
+        return regexes.every((regex) => regex.test(info));
```

Comment:
Don't think it is necessary to have a method on its own now. Just inline it? Same as `someMatch()`.